### PR TITLE
LibWeb: Don't crash on transformed clip-paths

### PR DESCRIPTION
**Pull request by Gingeh**
Fixes crashes when an element has a css transformation and a clip-path (I noticed this crash the most on some github pages).

The crash was occurring because `PaintableBox::get_masking_area()` returned null if the box had a transform because we supposedly didn't "Support clip-paths with transforms".

Here is a test page showing that transformed clip-paths already work perfectly:
transformed clip-paths

### CODE REVIEW

File: Userland/Libraries/LibWeb/Painting/PaintableBox.cpp [hunk 1]
```diff
@@ -1010,9 +1010,6 @@ void PaintableBox::set_needs_display(InvalidateDisplayList should_invalidate_dis
```
Comment (gmta) [inline Userland/Libraries/LibWeb/Painting/PaintableBox.cpp:1010]: Please put the very relevant info from your PR description in the commit message for future reference! :)

Comment (Gingeh) [reply Userland/Libraries/LibWeb/Painting/PaintableBox.cpp:1010]: good idea, updated it :+1:

Comment (Gingeh) [reply Userland/Libraries/LibWeb/Painting/PaintableBox.cpp:1010]: oh, extra lines are still counted towards the commit message length cap :/

Comment (awesomekling) [reply Userland/Libraries/LibWeb/Painting/PaintableBox.cpp:1010]: There is no commit message length cap, only a line length cap. :)

**Comment by awesomekling** [comment]
Can we add a test proving that this doesn't crash? :)

**Comment by Gingeh** [comment]
> Can we add a test proving that this doesn't crash? :)

done, added the demo in the screenshot above as a test

**Review by tcl3 — APPROVED** [review]

**Pull request merged**